decidim/decidim
# Refactor initiative signing wizard

#### :tophat: What? Why?
This aims to refactor the initiative signatures controller, so that it would not use the Wicked::Wizard gem anymore. After #10727, the wicked wizard is being used only by this controller, and makes sense to preserve the functionality while changing the underlying library. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #10727

#### Testing
1. Find an initiative that needs signing 
2. Configure the signing parameters (personal data collection, sms validation )
3. Attempt to sign it
4. See everything works as expected

:hearts: Thank you!
